projects/core/src/lazy-loading/facade-factory/facade-factory.service.ts

```diff
@@ -31,7 +39,8 @@ const PROXY_FACADE_INSTANCE_PROP = 'proxyFacadeInstance';
 export class FacadeFactoryService {
   constructor(
     protected featureModules: FeatureModulesService,
-    protected injector: Injector
+    protected injector: Injector,
```

Comment:
It seems that  `injector` in `protected injector: Injector,` is not used.

Reply:
Indeed.  I removed it to clean up the code.